### PR TITLE
docs(filesystem): include Google Drive into filesystem tutorial

### PR DESCRIPTION
**Pull request by IlyaFaer**
Towards https://github.com/dlt-hub/verified-sources/issues/308
Follows the changes made in the PR https://github.com/dlt-hub/dlt/pull/932

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ To get AWS keys for S3 access:
 For more info, see
 [AWS official documentation.](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
 
-#### Google Cloud Storage credentials
+#### Google Cloud Storage / Google Drive credentials
```
Comment (IlyaFaer) [inline docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md:51]: Google has unified credentials system for all their services, so it supposed to be all the same

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md [hunk 2]
```diff
@@ -133,11 +133,14 @@ For more information, read the
...
-   used, for example:
+   used:
 
    ```toml
    [sources.filesystem] # use [sources.readers.credentials] for the "readers" source
-   bucket_url="s3://my-bucket/csv_files/"
+   # bucket_url="az://my-bucket/csv_files/" - for Azure Blob Storage
+   # bucket_url="gdrive://my-bucket/csv_files/" - for Google Drive folder
+   # bucket_url="gs://my-bucket/csv_files/" - for Google Storage
+   bucket_url="s3://my-bucket/csv_files/" # for AWS S3
```
Comment (IlyaFaer) [inline docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md:143]: It'd be good to add every schema supported. There are examples in the doc, but not for all, so user, most likely, will have to verify what schema to set. But if they have a list of all supported schemas, they don't need to go anywhere else - the doc provides all.

Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md:143]: what do you mean by schema?

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md (hunk 1 above)
Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md:51]: makes sense

**Review by AstrakhantsevaAA — COMMENTED** [review]
All is good, but I didn't understand the question about schema

File: docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md (hunk 2 above)
Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md:143]: ah, you talking about this 
image

Comment (AstrakhantsevaAA) [reply docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md:143]: thank you! this is really helpful info

**Review by AstrakhantsevaAA — APPROVED** [review]

**Review by AstrakhantsevaAA — COMMENTED** [review]
- could you rename "google analytics API" to "Cloud Storage API" and "Google Drive API"
- also, we should give some info if user has no google cloud account, just regular google account and gdrive. In this case we can use OAuth.

if you don't want to waste your time on documentation, tell me, I'll reassign the task to someone else

**Comment by rudolfix** [comment]
@IlyaFaer google auth is supported. I was running tests on it. what is not supported is the python-data credential cache and browser authentication. but if you do that and get refresh token with proper scopes you can configure the thing to work. 

we are giving instructions on how to do it for many of our verified sources. some kind of guide on how to use `filesystem` with your personal gdrive and oauth would be cool! @AstrakhantsevaAA maybe we should give that task to @dat-a-man or someone else?

otherwise I'm good with this PR (I trust you guys :)

**Comment by IlyaFaer** [comment]
> what is not supported is the python-data credential cache and browser authentication. but if you do that and get refresh token with proper scopes you can configure the thing to work.

A-ah, so if I put personal account credentials into a file (instead of entering them in a web browser), it'll work? Okay, got it. Yeah, I'll add it.

**Review by AstrakhantsevaAA — APPROVED** [review]

**Comment by AstrakhantsevaAA** [comment]
reassigned to Aman

**Pull request merged**